db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java

```diff
@@ -212,13 +230,33 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
         }
     }
 
+    /**
+     * load properties file `sqlshell.rc`.
+     * when running from spliceengine, this will be the file in splice_machine/sqlshell.rc (because CWD is there)
+     * otherwise, just the file next to sqlshell.sh.
+     * @return properties from sqlshell.rc. on error: null
+     */
+    public static Properties getProperties() {
+        Properties prop = new Properties();
+        String fileName = "sqlshell.rc";
```

Comment:
Usually rc files start with a ".", e.g. .bashrc, .sqlshellrc
Should we do the same?

Reply:
done, renamed to `.sqlshellrc`